app/modules/ads/slots-loader.js

```diff
@@ -129,11 +129,14 @@ export const slotsLoader = {
   },
 
   loadFirstSlot() {
+    const { context } = window.Wikia.adEngine;
     const firstSlotName = `${this.baseSlotName}_1`;
+    const isDisabledOnScroll = context.get('slots.incontent_boxad_1.repeat.disablePushOnScroll');
```

Comment:
`loadFirstSlot` method seems to be general, it isn't used by `incontent_boxad_1` only. Why do we look for value on this particular slot here?

Reply:
it's the only repeatable slot atm and the function always calls for icb. Still a very good point so it will be generalized